src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,123 @@
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+import { Loader } from '../../components/Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PersonLink } from './PersonLink/PersonLink';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setLoading] = useState<boolean>(false);
```

Comment:
It's not necessary to explicitly specify the boolean type when using the useState hook. TypeScript is able to infer the type based on the initial value provided to useState. But it's not a mistake, just want you to know
```suggestion
  const [isLoading, setLoading] = useState(false);
```